hasura.planx.uk/migrations/1713084872473_create_view_teams_summary/up.sql

```diff
@@ -0,0 +1,26 @@
+CREATE OR REPLACE VIEW "public"."teams_summary" AS SELECT 
+    t.id,
+    t.name,
+    t.slug,
+    t.reference_code,
+    t.settings->>'homepage' as homepage,
+    t.domain as subdomain,
+    ti.has_planning_data as planning_data_enabled,
+    '@todo' as article_4s_enabled,    
+    t.notify_personalisation as govnotify_personalisation,
+    CASE 
+        WHEN coalesce(ti.production_govpay_secret, ti.staging_govpay_secret) is not null 
+        THEN true
+        ELSE false
+    END as govpay_enabled,
+    t.submission_email as send_to_email_address,
+    coalesce(ti.production_bops_submission_url, ti.staging_bops_submission_url) as bops_submission_url,
```

Comment:
Since we only sync `staging_` variables down to non-production environments, `coalesce()` is an easy way for making sure we display the right "done" per environment. 

Please note that if you want to check if a certain team has, for example, both a production & staging Gov Pay key configured - you'll have to check _both_ `editor.planx.uk/admin-panel` & `editor.planx.dev/admin-panel`. 

Having each environment ONLY reflect their own keys is the best way to ensure completeness in my opinion (eg because `teams.submission_email` for staging is actually only available on staging)